pandas/tests/indexes/object/test_indexing.py

```diff
@@ -156,59 +155,3 @@ def test_get_indexer_non_unique_np_nats(self, np_nat_fixture, np_nat_fixture2):
             expected_indexer = np.array([1, 3], dtype=np.intp)
             tm.assert_numpy_array_equal(indexer, expected_indexer)
             tm.assert_numpy_array_equal(missing, expected_missing)
-
-
-class TestSliceLocs:
```

Comment:
The remaining tests here were already parametrized with `any_string_dtype`, so only keeping them in `pandas/tests/indexes/string`